Packs/WootCloud/Integrations/WootCloud/WootCloud.yml

```diff
@@ -62,31 +62,35 @@ script:
       - name: date_range
         description: Examples are (2 hours, 4 minutes, 6 month, 1 day, etc.)
         isArray: false
-        default: true
+        default: false
+        defaultValue: '1 day'
```

Comment:
why did you add the default value?

Reply:
As I think that having a default value for arguments in general is a good idea.
But on a 2nd look I see that this arg is req.

Should I rm or keep?

Reply:
adding default value makes the `required` flag redundant as it will always have a value.
if you think that this value is a common case - leave it. otherwise, remove it.

Reply:
I see. will remove as i'm not familiar with the product enough to set a defaultvalue.